stubs/livewire/resources/views/profile/connected-accounts-form.blade.php

```diff
@@ -0,0 +1,70 @@
+<x-jet-action-section>
+    <x-slot name="title">
+        {{ __('Connected Accounts') }}
+    </x-slot>
+
+    <x-slot name="description">
+        {{ __('Manage and remove your connect accounts.') }}
+    </x-slot>
+
+    <x-slot name="content">
+        <h3 class="text-lg font-medium text-gray-900">
+            @if (count($this->accounts) == 0)
+                {{ __('You have no connected accounts.') }}
+            @else
+                {{ __('Your connected accounts.') }}
+            @endif
+        </h3>
+
+        <div class="mt-3 max-w-xl text-sm text-gray-600">
+            {{ __('You are free to connect any social accounts to your profile and may remove any connected accounts at any time. If you feel any of your connected accounts have been compromised, you should disconnect them immediately and change your password.') }}
+        </div>
+
+        <div class="mt-5 space-y-6">
+            @foreach ($this->providers as $provider => $enabled)
+                @if ($account = $this->accounts->where('provider_name', $provider)->first())
+                    <x-jet-connected-account provider="{{ $account->provider_name }}" created-at="{{ $account->created_at }}">
+
+                        <x-slot name="action">
+                            @if ($this->accounts->count() > 1)
```

Comment:
`@if (! is_null($user->password) || $this->accounts->count() > 1)`

Reply:
What does this resolve? This solution will have unwanted affects in that users who have signed up with OAuth, cannot connect to other providers?

Reply:
What does this resolve exactly? If I've signed up with Facebook, but later decided to connect Github and remove Facebook (and NOT set a password) this will not allow me to remove Facebook

Reply:
It should allow people who set a password to remove all their OAuth 2 accounts.

Reply:
I don't disagree, but I think that condition should be flipped so that users can remove any connected accounts, if they have more than one. An only be able to remove the last one if they've got a password set